daprdocs/content/en/developing-applications/building-blocks/state-management/state-store-ttl.md

```diff
@@ -58,6 +60,83 @@ dapr run --app-id orderprocessing --app-port 6001 --dapr-http-port 3601 --dapr-g
 
 {{% codetab %}}
 
+<!--dotnet-->
+
+```csharp
+// dependencies
+
+using Dapr.Client;
+
+// code
+
+await client.SaveStateAsync(storeName, stateKeyName, state, metadata: new Dictionary<string, string>() { 
+    { 
+        "metadata.ttlInSeconds", "120" 
+    } 
+});
+```
+
+To launch a Dapr sidecar and run the above example application, you'd then run a command similar to the following:
+
+```bash
+dapr run --app-id orderprocessing --app-port 6001 --dapr-http-port 3601 --dapr-grpc-port 60001 dotnet run
+```
+
+{{% /codetab %}}
+
+{{% codetab %}}
+
+<!--java-->
+
+```java
+//dependencies
+
+import io.dapr.client.DaprClient;
+import io.dapr.client.DaprClientBuilder;
+
+//code
+
+client.saveState(STATE_STORE_NAME, "order_1", Integer.toString(orderId)).block();
+client.saveState(STATE_STORE_NAME, "order_2", Integer.toString(orderId)).block();
```

Comment:
The Java example is not doing the same as in .Net.

Reply:
thanks for helping! I'll remove Java SDK for now